web3.go

```diff
@@ -49,6 +52,79 @@ func WeiAsGwei(w *big.Int) string {
 	return new(big.Rat).SetFrac(w, weiPerGwei).FloatString(9)
 }
 
+func CallConstantFunction(ctx context.Context, client Client, myabi abi.ABI, address, functionName string, parameters ...interface{}) (interface{}, error) {
+	var out interface{}
+	switch myabi.Methods[functionName].Outputs[0].Type.T {
+	case abi.BoolTy:
+		out = new(bool)
+	case abi.UintTy:
+		out = new(uint8)
```

Comment:
This will need to support up to 256 bits. It looks like `package abi` recognizes `big.Int`.